test/test.js

```diff
@@ -205,4 +205,12 @@ describe('scoped-require node module', function () {
 
     assert.deepEqual(baseModule.scopedDirs, [path.resolve(__dirname, 'scoped-dir')])
   })
+
```

Comment:
took me a while to see if we've missed something, but looks legit.
very elegant test